common/rpctypes/types.go

```diff
@@ -53,6 +53,7 @@ type MakeOfferRequest struct {
 	MinimumAmount float64            `json:"minimumAmount"`
 	MaximumAmount float64            `json:"maximumAmount"`
 	ExchangeRate  types.ExchangeRate `json:"exchangeRate"`
+	EthAsset      types.EthAsset     `json:"ethAsset"`
```

Comment:
```suggestion
	EthAsset      types.EthAsset     `json:"ethAsset,omitempty"`
```

this makes it optional, so that we can default to ETH if it's not set